README.rst

```diff
@@ -54,13 +54,12 @@ Traits has the following optional dependencies:
 
 * `NumPy <http://pypi.python.org/pypi/numpy>`_ to support the trait types
   for arrays.
-* `Traitsui <https://pypi.python.org/pypi/traitsui>`_ to support Gui
+* `TraitsUI <https://pypi.python.org/pypi/traitsui>`_ to support Gui
```

Comment:
nit : GUI instead of Gui? As long as this PR is about capitalization in README?

Reply:
Sure, let's do it!

Reply:
Addressed.